tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -781,6 +781,160 @@ int main(int argc, char **argv)
             }
         }
 
+        /* Test equal preference grouping */
+        {
+            /* Test server cipher preference negotiation for client-preferenced equal preference group */
```

Comment:
Can you write this comment in a way that is a little more readable?

Reply:
Ok